apps/game/lib/game/cup.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Cup do
+  @moduledoc """
+  generat a hand of n dices in a cup
```

Comment:
"generate a hand of n dice in a cup"

dice is the plural of die (the singular of "dice").